### ISSUE TITLE
Arrows/Buttons for moving section editor elements up or down or to top

### ISSUE THREAD

**Issue by HGustavs** [HGustavs/LenaSYS#7509]
![Screenshot 2019-05-24 12 46 07](https://user-images.githubusercontent.com/6978801/58322664-eb681c00-7e21-11e9-8c83-e30ac8aff1e2.png)

Buttons would function as follows:

* Add one tab
* Remove one tab
* Move up to top of current section
* Move to bottom of current section
* Move to top of all sections (or moment?)
* Move to bottom of all sections (or moment?)

**Comment by c17julka** [comment]
Since this would result in a lot of icons, a hamburger menu which contains these buttons could be implemented.

**Comment by a17ludca** [comment]
Moving up and down seems unnecessary since you can move a row easily by click and holding mousebutton. Adding tabs and removing tabs seems more appropriate.

**Comment by b17nicsv** [comment]
I agree with both a17ludca and c17julka. If all the buttons must be implemented then I highly suggest a hamburger menu of sorts to not clutter the item.

**Comment by a18jenro** [comment]
One should perhaps wait for this issue ( https://github.com/HGustavs/LenaSYS/issues/9172 )before starting on this one.
